apps/web/hooks/useFileUpload.ts

```diff
@@ -77,6 +85,9 @@ function useFileUpload() {
 
       if (!errors.length) {
         toast.success("All files has been processed!");
+        mutate(
+          `/api/files?page=${currentPage}&limit=${SAVED_FILES_PAGE_LIMIT}`,
```

Comment:
Why do you use `SAVED_FILES_PAGE_QUERY_KEY` in one place but ignore it in another place?